swagger.yaml

```diff
@@ -1,42 +1,281 @@
+        type: integer
+        example: 1
+      items:
+        type: array
+        items:
+          $ref: '#/definitions/Image'
+      limit:
+        description: "The number of images requested"
+        type: integer
+      offset:
+        description: "The first row of images to retrieve, starting at 0. Use this parameter as a pagination mechanism along with the limit parameter"
+        type: integer
+      total_count:
+        description: "The total number of images"
+        readOnly: true
+        type: integer
+        example: 1
+
+  Image:
+    type: object
+    description: "Image metadata model"
+    properties:
+      id:
+        type: string
+        description: "Image metadata unique identifier"
+        example: "042e216a-7822-4fa0-a3d6-e3f5248ffc35"
+      collection_id:
+        type: string
+        description: "Collection unique identifier corresponding to this image"
+        example: "5557dcd9-bf58-4a67-94f7-2343569834cc"
+      state:
+        type: string
+        description: |
+          The state of the dataset collection, can only be one of the following:
+          * created
+          * published
+        example: "published"
+      filename:
+        type: string
+        description: "image's file name"
+        example: "imageName.jpg"
+      license:
+        type: object
+        description: "license that defines the third party permissions for this image"
+        properties:
+          title:
+            type: string
+            description: "Title of the license"
+            example: "Open Government Licence v3.0"
+          href: 
+            type: string
+            description: "Link to the license content"
+            example: "https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+      original:
+        type: object
+        description: "original file upload in the upload bucket"
+        properties:
+          path:
+            type: string
+            description: "S3 object key (without bucket name) where the image is stored"
+            example: "imges/042e216a-7822-4fa0-a3d6-e3f5248ffc35/imageName.jpg"
+      downloads:
+        type: object
+        description: "all available downloads for this image keyed by resolutions, in WxH pixels. Note that this spec does not define all possible resolutions."
+        properties:
+          1920x1080:
+            $ref: '#/definitions/ImageDownload'
+          1280x720:
+            $ref: '#/definitions/ImageDownload'
+          
+  ImageDownload:
     type: object
+    description: "Download information for a particular image and resolution"
     properties:
-      message:
+      size:
+        type: integer
+        description: "File size in number of bytes"
+        example: 1024
+      href:
+        type: string
+        description: "Full URL pointing to the image in download service"
+        example: "http://dp-download-service/images/042e216a-7822-4fa0-a3d6-e3f5248ffc35/imageName.jpg"
```

Comment:
my understanding is that this is the URL in the download service to download the image, is that correct?